CONTRIBUTING.md

```diff
@@ -132,29 +135,22 @@ For RPMs, use `rpm -q pmdk` to display the name.
 For Deb packages, run `dpkg-query -W pmdk` and use the
 second (version) string.
 
-#### Milestone field
+#### Priority
 
-Optionally, assign the milestone the issue needs to be fixed before.
+Requested priority describes the urgency to resolve a defect and establishes
+the time frame for providing a verified resolution. Priorities are defined as:
 
-#### Type: Bug label
+* **P1**: Showstopper bug, requiring a resolution before the next release of the
+library.
+* **P2**: High-priority bug, requiring a resolution although it may be decided
+that the bug does not prevent the next release of the library.
+* **P3**: Medium-priority bug.  The expectation is that the bug will be
+evaluated and a plan will be made for when the bug will be resolved.
+* **P4**: Low-priority bug, the least urgent.  Fixed when the resources are available.
 
-Assign the `Type: Bug` label to the issue
-(see [GitHub Help](https://help.github.com/articles/applying-labels-to-issues-and-pull-requests) for details).
+### Other issues
 
-#### Priority label
-
-Optionally, assign one of the Priority labels (P1, P2, ...).
-The Priority attribute describes the urgency to resolve a defect
-and establishes the time frame for providing a verified resolution.
-These Priority labels are defined as:
-
-* **P1**: Showstopper bug, requiring resolution before the next release of the library.
-* **P2**: High-priority bug, requiring resolution although it may be decided that the bug does not prevent the next release of the library.
-* **P3**: Medium-priority bug.  The expectation is that the bug will be evaluated and a plan will be made for when the bug will be resolved.
-* **P4**: Low-priority bug, the least urgent.  Fixed as resources are available.
-
-Then describe the bug in the comment fields.
-
-#### Type: Feature label
-
-Assign the `Type: Feature` label to the issue, then describe the feature request in comment fields.
+On our issues page we also gather feature requests and questions. Templates to use
+are accordingly `Feature` and `Question`.  They should help deliver a meaningful
+description of a feature or ask a question to us (remember though we have
+different means of communication, as described on top of the page).
```

Comment:
on top of the page->at the top of the page?

Reply:
done